src/styles/_catalog.scss

```
@@ -1,22 +1,39 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$cardWidth: 200px;
+$cardGap: 48px;
 
-  max-width: 944px;
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+  gap: $cardGap;
+  justify-content: center;
   margin: 0 auto;
 
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, minmax($cardWidth, 1fr));
+    max-width: 4 * $cardWidth + 3 * $cardGap;
```

you don't need max-width here
just use power of the grid)
```suggestion
    grid-template-columns: repeat(4, $cardWidth);
```

---

fix in other places